src/state-schemas/state-schema.js

```diff
@@ -102,6 +102,20 @@ export const stateProperties = {
     type: "integer",
     default: 0
   },
+  editorCursorLine: {
+    type: "integer",
+    default: 0
+  },
+  editorCursorChar: {
+    type: "integer",
+    default: 0
+  },
+  editorCursorForceUpdate: {
+    // if this is true when the editor recieves it as props,
+    // then the editor must reposition the cursor using internal editor APIs
+    type: "boolean",
+    default: false
+  },
```

Comment:
I wonder if it would be better to just have an `editor` object that we can stuff properties into.